contracts/src/ethereum/TokenBridge.sol

```diff
@@ -79,17 +79,17 @@ contract TokenBridge is
         l2TokenBridge = l2TokenBridge_;
     }
 
-    modifier isValidL2Address(uint256 l2Address) {
+    modifier isValiduint256(uint256 l2Address) {
```

Comment:
Why the name change? `isValidL2Address` was a better name, `isValidFelt` is probably the best option

Reply:
Think this is a rebase artifact